src/components/common/Accordion/Accordion.interface.ts

```diff
@@ -0,0 +1,30 @@
+import { ReactNode } from 'react';
+import { CSSProperties } from 'styled-components';
+
+export interface AccordionItem {
+  title: string;
+  desc: string;
+  customExpandIcon?: {
+    open: ReactNode;
+    close: ReactNode;
+  };
+  subtitle?: string;
+  isDisabled?: boolean;
+}
+
+interface AdditionalProps {
+  expandIconPosition?: 'right' | 'left' | 'hide';
+  subtitlePosition?: 'title' | 'desc' | 'none';
+  hideExpandIcon?: boolean;
```

Comment:
expandIconPosition 에 hide 가 있고 `hideExpandIcon?: boolean;` 속성이 있는데 동일한 역할을 하는게 중복되어서 처리되는거 같아요!

```suggestion
  expandIconPosition?: 'right' | 'left';
  subtitlePosition?: 'title' | 'desc' | 'none';
  hideExpandIcon?: boolean;
```
이렇게 처리해도 괜찮을거 같아요

Reply:
hideExpandIcon을 뒤늦게 추가하다보니 Position에 이미 있다는 걸 깜빡했네요😅
알려주셔서 감사합니다! 말씀해주신걸로 수정할게요